types/bun/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false,
+        "unified-signatures": false
+    }
```

Comment:
No need for this file anymore; these rules are not enabled (and tslint is one PR from going away).